stingray/deadtime/fad.py

```diff
@@ -0,0 +1,189 @@
+import warnings
+import numpy as np
+import scipy
+import matplotlib.pyplot as plt
+
+from scipy.ndimage.filters import gaussian_filter1d
+from scipy.interpolate import UnivariateSpline
+from astropy import log
+from astropy.table import Table
+
+from ..gti import cross_two_gtis, bin_intervals_from_gtis
+
+
+def _get_fourier_intv(lc, start_ind, end_ind):
+    time = lc.time[start_ind:end_ind]
+    counts = lc.counts[start_ind:end_ind]
+
+    fourier = scipy.fftpack.fft(counts)
+
+    freq = scipy.fftpack.fftfreq(len(time), lc.dt)
+    good = freq > 0
+
+    return freq[good], fourier[good], fourier[good] * np.sqrt(2 / (np.sum(counts)))
+
+
+def calculate_FAD_correction(lc1, lc2, segment_size, gti=None,
+                             plot=False, smoothing_alg='gauss',
+                             smoothing_length=None, verbose=False, tolerance=0.05,
+                             strict=False, all_leahy=False):
+    """Calculate Frequency Amplitude Difference-corrected (cross) power spectra.
+
+    Reference: Bachetti \& Huppenkothen, 2018, ApJ, 853L, 21
+
+    Parameters
+    ----------
+    lc1: class:`stingray.ligthtcurve.Lightcurve`
```

Comment:
The light curves need not be _identical_, but surely the flux must be similar. Also, it cannot apply to light curves in different energy bins from the same instrument, because dead time would not be independent. I modified the docstring accordingly.